src/heroic.rs

```diff
@@ -227,10 +230,22 @@ impl HeroicGames {
             self.games
                 .insert((root.clone(), official.clone()), MemorizedGame { install_dir, prefix });
         } else {
-            log::info!(
-                "memorize_game did not find {} in manifest, no backup/restore will be done!",
-                title
-            );
+            // Handling game name mismatches, e.g. GRIP vs. GRIP: Combat Racing
+            if self.normalized_to_official.len() == 1 {
+                // user clicked on a game in the UI which call us with this single game
+                log::trace!(
+                        "heroic::memorize_game did not find neither '{}' nor '{}' in ludusavi manifest, no backup/restore can be done!  User probably selected a single, different game for backup/restore.",
+                        title, normalized
+                    );
+            } else if std::env::var("LUDUSAVI_DEBUG").is_ok() {
+                eprintln!(
+                    "Ignoring unrecognized Heroic game: '{}' (normalized: '{}')",
+                    title, normalized
+                );
+            }
+            log::info!("heroic::memorize_game did not find neither '{}' nor '{}' in ludusavi manifest, no backup/restore can be done!",
+                        title, normalized);
```

Comment:
Since this is outside of the if-else, it'll make a duplicate log if `self.normalized_to_official.len() == 1` triggered. No worries, though; I'll update it post-merge.